.github/workflows/starknet-rpc-tests.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: Task - Rpc Tests
+
+on:
+  workflow_dispatch:
+  workflow_call:
+
+jobs:
+  rpc-tests:
+    runs-on: ubuntu-latest
+    env:
+      BINARY_PATH: ../target/release/madara
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key:
+            ${{ runner.os }}-cargo-${{ hashFiles('Cargo.lock') }}-${{
+            github.run_id }}
+          fail-on-cache-miss: true
+      - name: Setup build deps
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y clang llvm libudev-dev protobuf-compiler
+      - name: Run rpc test
+        run: |-
+          ./target/release/madara setup
+          ./target/release/madara run --dev --alice --sealing=manual --execution=Native --tmp &
```

Comment:
`--alice` is enabled by default with the `--dev` flag, u can remove it. Using `--execution native` might speed up the tests, but at the cost that we don't detect errors on the wasm

Reply:
> --alice

Will do

> native

we can do two rounds of tests, their are fast to execute anyway, on in native, one in wasm

Reply:
done